checker/src/main.rs

```diff
@@ -40,12 +41,15 @@ fn main() {
         env_logger::init_from_env(e);
     }
 
-    // Configure the analysis to just run on a single function if set
-    let analyze_single_func = if env::var("SINGLE_FUNC").is_ok() {
-        env::var("SINGLE_FUNC").ok()
+    // Configure the analysis target
+    let target = if env::var("SINGLE_FUNC").is_ok() {
```

Comment:
I should not have let SINGLE_FUNC go by me, but now seems a time to call a halt: We should not keep inventing environmental variable names that are not prefixed with MIRAI and should fix this one too. Generic names that might clash with names used by other tools create an adoption barrier for MIRAI.

Reply:
Introduced now an argument parser based on the clap crate.